src/main/java/com/google/api/codegen/viewmodel/InitCodeView.java

```diff
@@ -24,6 +26,21 @@
 
   public abstract List<FieldSettingView> fieldSettings();
 
+  /**
+   * A ModelTypeTable used to hold information about the types used in method samples.
+   * This is commonly used for type aliasing.
+   */
+  @Nullable
+  public abstract ModelTypeTable typeTable();
```

Comment:
A view should only hold static information, it should not have any logic classes in it.